docs/CLI.md

```diff
@@ -318,6 +318,26 @@ If configuration files are found in the specified paths, _all_ projects specifie
 
 :::
 
+### `--randomize`
+
+Shuffle the order of the tests within a file. The seed used to generate the new order will be printed to stdout.
```

Comment:
Add something along the lines of "passing the same `seed` will allow you to recreate the test run" or some such? Preferably with a better wording 😅